docs/contribute.md

```diff
@@ -0,0 +1,157 @@
+
+### Building the documentation
+
+To verify how your changes look, generate the static site with the documentation. This process is called *building*. You can do it in these ways:
+- [use Docker](#use-docker)
+- [build locally](#build-locally)
+
+#### Use Docker
+
+1. [Get Docker](https://docs.docker.com/get-docker/)
+2. We use [our Docker image](https://hub.docker.com/repository/docker/perconalab/pmm-doc-md) to build documentation. Run the following command:
+
+```sh
+docker run --rm -v $(pwd):/docs perconalab/pmm-doc-md mkdocs build
+```
+   If Docker can't find the image locally, it first downloads the image, and then runs it to build the documentation.
+
+3. Go to the ``site/`` directory and open the ``index.html`` file to see the documentation.
+
+   If you want to see the changes as you edit the docs, use this command instead:
+
+   ```sh
+   docker run --rm -v $(pwd):/docs -p 8000:8000 perconalab/pmm-doc-md mkdocs serve --dev-addr=0.0.0.0:8000
+   ```
+
+   Wait until you see the message `INFO    -  Start detecting changes`, then enter `0.0.0.0:8000` in the browser's address bar. The documentation automatically reloads after you save the changes in source files.
+
+
+#### Build locally
+
+1. Install [pip](https://pip.pypa.io/en/stable/installing/)
+2. Install [MkDocs].
+3. While in the root directory of the doc project, run the following command to build the documentation:
+
+```sh
+mkdocs build
+```
+4. Go to the ``site`` directory and open the ``index.html`` file in your web browser to see the documentation.
+5. To automatically rebuild the documentation and reload the browser as you make changes, run the following:
+   
+   ```sh
+   mkdocs serve
+   ```
+   Wait until you see the message `INFO    -  Start detecting changes`, then enter `0.0.0.0:8000` in the browser's address bar. 
+
+## PDF
+
+To create the PDF version of the documentation, use the following command:
+
+* With Docker:
+
+    ```sh
+    docker run --rm -v $(pwd):/docs -e ENABLE_PDF_EXPORT=1 perconalab/pmm-doc-md mkdocs build -f mkdocs-pdf.yml
+    ```
+
+* Without:
+
+    ```sh
+    ENABLE_PDF_EXPORT=1 mkdocs build -f mkdocs-pdf.yml
+    ```
+
+The PDF is in `site/_pdf`.
+
+## Repository structure
+
+The repository includes the following directories and files:
+
+- `mkdocs-base.yml` - the base configuration file. It includes general settings and documentation structure.
+- `mkdocs.yml` - configuration file. Contains the settings for building the docs with Material theme.
+- `mkdocs-pdf.yml` - configuration file. Contains the settings for building the PDF docs.
+- `docs`:
+  - `*.md` - Source markdown files.
+  - `_images` - Images, logos and favicons
+  - `css` - Styles
+  - `js` - Javascript files
+- `_resource`:
+   - `templates`:
+     - ``styles.scss`` - Styling for PDF documents
+   - `theme`:
+      - `main.html` - The layout template for hosting the documentation on Percona website
+   - overrides - The folder with the customized templates
```

Comment:
Check the structure. You do not have some folders in a project, so do not mention them here. Also, the list is broken.